src/aelfrice/hook_tail.py

```
@@ -0,0 +1,336 @@
+    beliefs_obj = record.get("beliefs")
+    beliefs: list[dict[str, object]] = []
+    if isinstance(beliefs_obj, list):
+        beliefs = [b for b in beliefs_obj if isinstance(b, dict)]
+    n_l0 = sum(1 for b in beliefs if b.get("lane") == "L0")
+    n_l1 = sum(1 for b in beliefs if b.get("lane") == "L1")
+    parts: list[str] = [short_ts, hook]
+    if isinstance(tokens, int):
+        parts.append(f"{tokens} tok")
+    if isinstance(latency_ms, int):
+        parts.append(f"{latency_ms} ms")
+    parts.append(f"L0×{n_l0} L1×{n_l1}")
```

Ruff (0.15.12)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)